src/setup.js

```diff
@@ -136,11 +136,6 @@ function configure({ env, stage, stageUrl, apiStage, localhostUrl, checkoutUri,
         config.apiStage = apiStage;
     }
 
-    localhostUrl = localhostUrl || (Button.xprops && Button.xprops.localhostUrl) || (Checkout.xprops && Checkout.xprops.localhostUrl);
```

Comment:
Looks good to me! Let's also delete this `localhostUrl` from the props too. We shouldn't need it at all.

```
src/button/component.jsx
```